dat/ships/zalek/zalek_demon_type_e.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<ship inherits="Za'lek Demon" name="Za'lek Demon Class E">
```

Comment:
Maybe call it `Za'lek Demon Class IV or something like that` for consistency with the other Za'lek variants?